### PR TITLE
Task1 part1 structure layout

### PR DESCRIPTION
**Pull request by Mohsen-Awad**

### CODE REVIEW

File: task_1/index.html [hunk 1]
```diff
@@ -1,13 +1,48 @@
...
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+        </div>
+    </div>
+
 </body>
+
 </html>
```
Comment (laithyounes) [inline task_1/index.html:48]: please add newline at the end of the file 
ref: https://stackoverflow.com/a/5813359

File: task_1/style.css [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+
+.grid-items {
+    display: grid;
+    border: 1px solid;
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+    margin-top: 20px;
+    grid-gap: 32px;
+    grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
 }
```
Comment (laithyounes) [inline task_1/style.css:60]: please add newline at the end of the file 
ref: https://stackoverflow.com/a/5813359

File: task_1/style.css [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+
+.wrapper {
+    margin: 0 auto;
+    max-width: 1440px;
+}
+
+.header {
+    height: 80px;
+    width: 100%;
+    border: 1px solid;
```
Comment (laithyounes) [inline task_1/style.css:14]: let's sort the CSS alphabetically

File: task_1/style.css [hunk 4]
```diff
@@ -1,3 +1,60 @@
...
+    margin: auto;
+    border: 1px solid;
+    width: 217px;
+    height: 400px;
+}
+
+.grid-items {
+    display: grid;
+    border: 1px solid;
+    padding-top: 3rem;
```
Comment (laithyounes) [inline task_1/style.css:55]: use shorthand padding

File: task_1/index.html [hunk 5]
```diff
@@ -1,13 +1,48 @@
...
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    <script src="index.js"></script>
+    <!-- <script src="index.js"></script> -->
```
Comment (laithyounes) [inline task_1/index.html:13]: please move the script tag to the bottom

**Review by laithyounes — CHANGES_REQUESTED** [review]
please apply the requested changes, other than that LGTM

File: task_1/style.css (hunk 3 above)
Comment (abood454) [reply task_1/style.css:14]: let's sort the CSS alphabetically

File: task_1/index.html (hunk 1 above)
Comment (araraltawarah98) [inline task_1/index.html:48]: add newline at the end of the file

**Review by araraltawarah98 — APPROVED** [review]

**Review by abood454 — APPROVED** [review]
good job

File: task_1/index.html [hunk 6]
```diff
@@ -1,13 +1,48 @@
...
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+        </div>
+    </div>
+
 </body>
```
Comment (ashrafabuodeh) [inline task_1/index.html:46]: add new line

**Review by ashrafabuodeh — CHANGES_REQUESTED** [review]

**Review by laithyounes — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

**Review by abood454 — APPROVED** [review]

**Review by ashrafabuodeh — APPROVED** [review]

**Pull request merged**